src/Dungeon/DungeonGenerator.elm

```diff
@@ -323,10 +323,10 @@ findAndUpdateRoom targetRoom ({ rooms, activePoints } as model) =
                 someOtherPoint ->
                     someOtherPoint :: updatedPoints
 
-        activePoints' =
+        activePoints_ =
```

Comment:
I guess their hope was that people would give things semantic names (`modifiedActivePoints` or `newActivePoints`) rather than just using the same name with a punctuation symbol after it.